timesketch/lib/analyzers/contrib/misp_analyzer.py

```diff
@@ -28,29 +28,29 @@ def __init__(self, index_name, sketch_id, timeline_id=None, **kwargs):
             timeline_id: The ID of the timeline.
         """
         super().__init__(index_name, sketch_id, timeline_id=timeline_id)
-        self.misp_url = kwargs.get("misp_url")
-        self.misp_api_key = kwargs.get("misp_api_key")
+        self.misp_url = current_app.config.get("MISP_URL")
+        self.misp_api_key = current_app.config.get("MISP_API_KEY")
         self.total_event_counter = 0
-        self.request_set = set()
         self.result_dict = dict()
+        self._query_string = kwargs.get("query")[0]
+        self._attr = kwargs.get("query")[1]
+        self._timesketch_attr = kwargs.get("query")[2]
```

Comment:
Optional suggestion to make it easier to read the code: 

```suggestion
        self._query_string = kwargs.get("query_string")
        self._attr = kwargs.get("attr")
        self._timesketch_attr = kwargs.get("timesketch_attr")
```